src/modules/infobox/components/FeatureListCategoryItem.vue

```diff
@@ -74,19 +79,43 @@ function showContentAndScrollIntoView(event) {
         return false
     }
 }
+
+onMounted(() => {
+    // there is no easy way to see if a item is truncated by css, so we
+    // check if the offset width is smaller than the scroll width.
+    // when there is a truncate, the offset changes but not the scroll, which
+    // means we can use that to detect truncating.
+    // this might be a bit hacky, but it works :)
+    if (featureTitle.value.offsetWidth < featureTitle.value.scrollWidth) {
+        tippyTitle = tippy(featureTitle.value, {
+            content: name.value,
+            hideOnClick: true,
+            placement: 'top',
+            delay: [500],
+            // Show tippy on long touch for mobile device
+            touch: ['hold', 500], // 500ms delay,
+        })
+    }
+})
+onUnmounted(() => {
+    tippyTitle?.destroy()
+})
```

Comment:
The component TextTruncate already handle this, so better to reuse it

Reply:
See https://github.com/geoadmin/web-mapviewer/blob/876902fbd0f8a206a8320f74f5e1bce6e23e5ead/src/utils/components/TextTruncate.vue#L1

Reply:
Hoo, Pascal told me about it but I thought he meant the `text-truncate` bootstrap component, not the vue component. That'll simplify things.